test/custom/TokenGating.test.js

```diff
@@ -16,7 +16,7 @@ const {
 const { placeBid } = require('../helpers/externalTransactions');
 const { deployTestContractSetup, deploy, getTokenVault } = require('../helpers/deploy');
 const { MARKETS, FOURTY_EIGHT_HOURS_IN_SECONDS } = require('../helpers/constants');
-const { testCases } = require('../testCases.json');
+const { testCases } = require('../partyBidTestCases.json');
```

Comment:
partybuy tests are in PartyBuy folder, but partybid tests are in root folder - should both be in sub-folders?